scripts/deploy.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+SHA="$(git rev-parse HEAD)"
+DEVCYCLE_PROD_SLEUTH_API_TOKEN="$(aws secretsmanager get-secret-value --secret-id=DEVCYCLE_PROD_SLEUTH_API_TOKEN | jq -r .SecretString )"
```

Comment:
We don't grant access to aws for public repos, you'll have to set this as a github secret

Reply:
https://github.com/DevCycleHQ/js-sdks/blob/main/scripts/npm-safe-publish.sh#L46

How's that different to this? They're both deployed locally and manually.

Reply:
ah; sorry - thought this was a github action